doc/swap_state_machine.txt

```diff
@@ -0,0 +1,19 @@
+digraph finite_state_machine {
+	fontname="Helvetica,Arial,sans-serif"
+	node [fontname="Helvetica,Arial,sans-serif"]
+	edge [fontname="Helvetica,Arial,sans-serif"]
+	rankdir=LR;
+	node [shape = doublecircle]; FinishA FinishB;
+	node [shape = circle];
+	# Alice
+	StartA -> CommitA [label = "Request::InitSwap"];
+	CommitA -> RevealA [label = "Msg::Reveal"];
+	RevealA -> RefundsigA [label = "Msg::RefundProcedureSignatures"]
+	RefundsigA-> FinishA [label = "Event::TransactionConfirmations"]
+	
+	# Bob
+	StartB -> CommitB [label = "Request::InitSwap"];
+	CommitB -> RevealB [label = "Msg::Reveal"];
+	RevealB -> CorearbB [label = "Msg::CoreArbitratingSetup"];
+	CorearbB -> FinishB [label = "Event::TransactionConfirmations"]
```

Comment:
Buy sigs msg missing

Reply:
How did I miss this?? 
e876c3c8c1ccf61c14e2147d948743e875c5ba5e